### PR TITLE
Add support for MESOS_HTTP and MESOS_HTTPS healthchecks

### PR DESCRIPTION
**Pull request by philipnrmn**
Marathon now supports passing HTTP and HTTPS healthchecks through to Mesos. At the API level, this is simply a matter of specifying `"protocol": "MESOS_HTTP"` instead of `"protocol": "HTTP"` when posting to `/v2/apps`. 

At the UI level, we have swapped out the Marathon HTTP and HTTPS healthchecks formerly added when the user selected HTTP from the healthcheck type dropdown with the Mesos equivalent. The Marathon Command healthcheck is still available. 

Longform moving picture available here:
https://cl.ly/1H341o0O3z1t

### CODE REVIEW

**Comment by philipnrmn** [comment]
Retest this please

**Comment by rcorral** [comment]
retest this please

File: plugins/services/src/js/utils/__tests__/HealthCheckUtil-test.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ describe('HealthCheckUtil', function () {
     });
 
     it('should return false for a unknown protocol', function () {
-      expect(HealthCheckUtil.isKnownProtocol('MESOS_HTTP')).toEqual(false);
+      expect(HealthCheckUtil.isKnownProtocol('MESOS_GOPHER')).toEqual(false);
```
Comment (nLight) [inline plugins/services/src/js/utils/__tests__/HealthCheckUtil-test.js]: GOPHER 👍

**Comment by nLight** [comment]
Looks good!

**Comment by rcorral** [comment]
@philipnrmn could you rebase please?

**Comment by jfurrow** [comment]
Looks good, works great!

**Pull request merged**